src/components/BoardResolution/index.module.scss

```
@@ -0,0 +1,150 @@
+.containerBoard {
+  padding: 31.5px 42.5px;
+  border: 1px solid #E1E1E2;
+  position: relative;
+  border-radius: 4px;
+  max-height: 374px;
+  max-width: 395px;
+  margin-top: 10px;
+}
+
+.legend {
+  position: absolute;
+  top: -35px;
+  left: 10px;
+  background-color: white;
+  padding: 0 5px;
+  color: #525256;
+  font-size: 24px;
+  font-weight: lighter;
+}
+
+.board {
+  background-color: #173040;
+  height: 311px;
+  width: 311px;
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  padding: 15px 14px;
+  border-radius: 3.66px;
+}
+
+.cell {
+  width: 25%;
+  height: 25%;
+}
+
+.cell:nth-child(odd) {
+  background-color: #FEC34D;
+}
```

Aquí podrías utilizar la sintaxis de sass

```suggestion
.cell {
  width: 25%;
  height: 25%;

  &:nth-child(odd) {
    background-color: #FEC34D;
  }
}
```

>